v2/emailpassword/nextjs/app-directory/protecting-route.mdx

```diff
@@ -240,7 +240,7 @@ export async function HomePage() {
              * This means that the session does not exist but we have session tokens for the user. In this case
              * the `TryRefreshComponent` will try to refresh the session.
              */
-            return <TryRefreshComponent />;
+            return <TryRefreshComponent key={Date.now()} />;
```

Comment:
add comment explaining why this is needed

Reply:
done